### ISSUE TITLE
reference-style links not working

### ISSUE THREAD

**Issue by longavailable** [jeffreytse/jekyll-spaceship#43]
[Reference-style links](https://markdown-guide.readthedocs.io/en/latest/basics.html#links-reference) wouldn't work after the installation of [jekyll-spaceship](https://github.com/jeffreytse/jekyll-spaceship). 

I used the following syntax in the table of a jekyll post:
```
...
|  [text] | ||
...

[text]: links
```

It can not work now. It will display `[text]` as the output. Any suggestion?

**Comment by jeffreytse** [comment]
Hi @longavailable 

Thanks for the reporting. I will focus on this issue and welcome to star this project for the updates in the future.

Thanks and regards

**Comment by jeffreytse** [comment]
Hi @longavailable, I think you write a wrong reference-style syntax, the example is as below:

```md
...
|  [text][ref] | ||
...

[ref]: links
```

**Comment by longavailable** [comment]
Hi @jeffreytse They are just two different forms. [example]

`example` was used the syntax I mentioned.

[example]: https://markdown-it.github.io/#md3=%7B%22source%22%3A%22%5Bjekyll-spaceship%5D%5Cn%5Cn%5Cn%5Bjekyll-spaceship%5D%5B1%5D%5Cn%5Cn%5Cn%5Cn%5Bjekyll-spaceship%5D%3A%20https%3A%2F%2Fgithub.com%2Fjeffreytse%2Fjekyll-spaceship%5Cn%5B1%5D%3A%20https%3A%2F%2Fgithub.com%2Fjeffreytse%2Fjekyll-spaceship%22%2C%22defaults%22%3A%7B%22html%22%3Afalse%2C%22xhtmlOut%22%3Afalse%2C%22breaks%22%3Afalse%2C%22langPrefix%22%3A%22language-%22%2C%22linkify%22%3Atrue%2C%22typographer%22%3Atrue%2C%22_highlight%22%3Atrue%2C%22_strict%22%3Afalse%2C%22_view%22%3A%22html%22%7D%7D

**Comment by jeffreytse** [comment]
Hi @longavailable 

Thanks for the exampe, I will focus on this issue and fix it as soon as possible.

Thanks and regards

**Comment by longavailable** [comment]
And both of them are not working when `jekyll-spip` is activated. 

```
| **Applications** ||
| ---------------------------|----------------------------|
| [jekyll-spaceship]        | |
| [jekyll-spaceship][1]        | |

[jekyll-spaceship]: https://github.com/jeffreytse/jekyll-spaceship
[1]: https://github.com/jeffreytse/jekyll-spaceship
```

It would like the following. 

![image](https://user-images.githubusercontent.com/24582280/111598579-5611f500-880a-11eb-85ea-c100d07e65f5.png)

**Comment by jeffreytse** [comment]
Hi @longavailable 

Thanks for the reporting and this issue has been address and fixed. Please update your plugin with the repository's master branch in the `Gemfile` as below snippet.

```ruby
gem 'jekyll-spaceship', git: 'https://github.com/jeffreytse/jekyll-spaceship'
```

Welcome to star this project for further udpates in the future. : )

Thanks and regards

**Comment by longavailable** [comment]
Thank you very much for your quick response. It works.

Well, I have to report an advanced bug.

```
| **Windows Application** ||
| ---------------------------|----------------------------|
| Application / Softs        | Download links / Webpages  |
|[jekyll-spaceship] ***** [jekyll-spaceship2]||

[jekyll-spaceship]: https://github.com/jeffreytse/jekyll-spaceship
[jekyll-spaceship2]: https://github.com/jeffreytse/jekyll-spaceship
```

The second link was not working. 

![image](https://user-images.githubusercontent.com/24582280/111649006-ec600e00-883e-11eb-99ab-fc12b8dc3875.png)

**Comment by jeffreytse** [comment]
Hi @longavailable

Thanks for the advanced reporting, now this issue has been addressed and fixed, please update your plugin to the latest version.

Thanks and regards

**Comment by jeffreytse** [comment]
Hi @longavailable , Could we close this issue? Welcome to get back if you have any issue. : )

**Comment by longavailable** [comment]
I‘ll close it.

**Comment by longavailable** [comment]
One more error:

```
| **Windows Application** ||
| ---------------------------|----------------------------|
| Application / Softs        | Download links / Webpages  |
|[jekyll-spaceship (JS)] | |

[jekyll-spaceship (JS)]: https://github.com/jeffreytse/jekyll-spaceship
```

It got the following:

![image](https://user-images.githubusercontent.com/24582280/112089277-49a3e880-8bcc-11eb-8ebc-2a6bd6b0e717.png)

**Comment by jeffreytse** [comment]
Hi @longavailable 

Thanks for the advanced reporting, now this issue has been addressed and fixed, please update your plugin to the latest version.

Thanks and regards

**Comment by longavailable** [comment]
I suppose the following problem is related. So I post it here.

I add to two urls within angle brackets to make them as links. It works fine at a normal single cell. While it cannot be turned successfully for a url after the `^^` symbol. Please check my case:

```
| **Windows Application** ||
| ---------------------------|----------------------------|
| Application / Softs        | Download links / Webpages  |
|jekyll-spaceship | <https://github.com/jeffreytse/jekyll-spaceship> |
|^^ |^^ <https://github.com/jeffreytse/jekyll-spaceship> |
```

The second url can not be turned into a hyperlink.

![image](https://user-images.githubusercontent.com/24582280/112405941-3751b800-8d4e-11eb-9219-2e19d5446e99.png)

**Comment by jeffreytse** [comment]
Hi @longavailable 

Thanks for the advanced reporting, now it has been addressed and fixed on the issue #47, please update your plugin to the latest version.

Thanks and regards
